deps.ts

```diff
@@ -4,8 +4,8 @@ export {
   fromFileUrl,
   resolve,
   toFileUrl,
-} from "https://deno.land/std@0.173.0/path/mod.ts";
-export { basename, extname } from "https://deno.land/std@0.173.0/path/mod.ts";
+} from "https://deno.land/std@0.178.0/path/mod.ts";
+export { basename, extname } from "https://deno.land/std@0.178.0/path/mod.ts";
```

Comment:
These two `path` exports use the same specifier and can be merged into one statement.